File: assignment 10 - C# practice problems/30.cs
//Write a c# function to clone an array.
using System;
namespace MyNameSpace
{
    class Program30
    {
        public static void CloneArrayDriverFunction()
        {

            var numArray = new[] { 1, 2, 3, 4, 5 };
            var clonedArray = cloneArray(numArray);

            Console.WriteLine("old array " + numArray);
            Console.WriteLine("cloned array " + clonedArray);

        }

        static int[] cloneArray(int[] arrayParam)
        {
            return (int[])arrayParam.Clone();
        }

    }
}

File: assignment 3 - js practice problems/web-dev-starter/30.js
//Write a JavaScript function to clone an array.
function cloneArray(arrayParam) {
    return arrayParam.map((x) => x)
}

function cloneArrayDriverFunction() {
    let numArray = [1, 2, 3, 4, 5];
    let clonedArray = cloneArray(numArray);

    console.log("old array " + numArray);
    console.log("cloned array " + clonedArray);
}

cloneArrayDriverFunction()